app/views/layouts/public.html.erb

```diff
@@ -5,7 +5,6 @@
     <main class="flex h-screen bg-white dark:bg-gray-800">
       <%= content_for?(:main) ? yield(:main) : yield %>
     </main>
-
```

Comment:
@olimart Do you have any principles on white space? I just mean: when you add blank lines and when you don’t? For example, the routes.rb file has some new lines separating sections and you left those, but you decided to remove this one.

I don’t have any clear principles so I just add blank lines when I feel like it visually helps me to grock a file. For example, I think I added this one because I thought it was too easy to visually miss the render `toast` inclusion. It was kind of visually lumping in with the “main” right above it, but they’re conceptually serving very different purposes. But I’m not principled about white space, I just add some here and remove others there and it seems like you may have some better thinking behind it than I do. :)

Reply:
@krschacht TBH I don't have principles either.

> I just add blank lines when I feel like it visually helps me to grock a file

same here 😆 
This linter exists but I've never used it before
https://github.com/Shopify/erb-lint

Reply:
:)

it doesn’t feel worth a linter

since we’re both just winging it, I’d probably leave this blank line to help separate the toast from the main. But I don’t feel strongly so I’m good with whichever way you want to go,